R/summary.R

```diff
@@ -44,14 +40,17 @@ summary.tsd_onset <- function(object, ...) {
   # Extract the time_interval
   time_interval <- attr(object, "time_interval")
 
-  # Extract the seasons
-  seasons <- toString(unique(object$season))
+  # Extract the season
+  last_season <- unique(last_observation$season)
```

Comment:
Is unique needed here?
```suggestion
  last_season <- last_observation$season
```